.nvmrc

```diff
@@ -0,0 +1 @@
+v8.11.4
```

Comment:
I know this PR was closed, but looks like [`.node-version`](https://github.com/desktop/desktop/blob/b7895b606c25b29cb735e3affce2ffd38a4870bd/.node-version) was missed in the update. Though, I'm not sure what the difference between that file, `.nvmrc`, and `.tool-versions` is for...

Reply:
thanks for the catch @streeter! i'll get that in another pr in just a minute

Reply:
https://github.com/desktop/desktop/pull/5705